TestingHomework-Discounts/Models/Id.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TestingHomework_Discounts.Models
+{
+    public struct Id : IComparable, IComparable<Id>, IEquatable<Id>
```

Comment:
What was the intent of including the Id type? I notice it is not used in any types

My intended use of it in the example project is to protect my code from changes in underlying id type